app/src/main/res/values-fr-rFR/strings.xml

```diff
@@ -123,10 +123,11 @@
     <string name="missing_GPS_Perm_Dialog_title">autorisation GPS manquante</string>
     <string name="missing_GPS_Perm_Dialog_text">Vous avez refusé l\'autorisation GPS. L\'application
     en a besoin pour fonctionner. Cliquez OUI puis autorisez ou cliquez annuler pour fermer l\'application</string>
-
+    <string name="internet_offline_msg">Vous êtes hors ligne</string>
+    <string name="background_protection_msg">Votre protection est encore activée</string>
```

Comment:
Isn't a bit misleading saying that the app is "protecting" the user?

Reply:
Agree, just marketing